Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/audit/transform/DocRetrieveAuditTransforms.java

```diff
@@ -410,9 +413,9 @@ protected ActiveParticipant getActiveParticipantDestination(NhinTargetSystemType
 
         participant.getRoleIDCode()
             .add(AuditDataTransformHelper.createCodeValueType(
-                    AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DEST, null,
-                    AuditTransformsConstants.ACTIVE_PARTICIPANT_CODE_SYSTEM_NAME,
-                    AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DESTINATION_DISPLAY_NAME));
+                AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DEST, null,
+                AuditTransformsConstants.ACTIVE_PARTICIPANT_CODE_SYSTEM_NAME,
+                AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DESTINATION_DISPLAY_NAME));
         return participant;
     }
 
```

Comment:
This is not part of this ticket but it could be an improvement.Can you fix this method to return null in case of "if" condition not satisfied? 
private String getDocumentUniqueIdFromResponse(RetrieveDocumentSetResponseType response) {
        if (response != null
            && response.getDocumentResponse() != null
            && response.getDocumentResponse().get(0) != null
            && response.getDocumentResponse().get(0).getDocumentUniqueId() != null) {
            return response.getDocumentResponse().get(0).getDocumentUniqueId();
        } else {
            return null;
        }
    }